decompiler/frontend/binaryninja/handlers/constants.py

```diff
@@ -30,8 +32,10 @@ def register(self):
             }
         )
 
-    def lift_constant(self, constant: mediumlevelil.MediumLevelILConst, **kwargs) -> Constant:
+    def lift_constant(self, constant: mediumlevelil.MediumLevelILConst, **kwargs) -> Union[NotUseableConstant, Constant]:
```

Comment:
a `NotUsableConstant` is also a `Constant`, so the union is not necessary here for the typing.

Reply:
Yes